.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish 📦 to TestPyPI
+
+on:
+  create:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish 📦 to TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+<<<<<<< HEAD
```

Comment:
You committed some unresolved merge conflict

Reply:
Oops, let me fix that.